app/components/primer/alpha/segmented_control/button.html.erb

```diff
@@ -0,0 +1,9 @@
+<% if @icon_only == :always %>
+  <%= render Primer::Beta::IconButton.new(icon: @icon, "aria-label": content, **@system_arguments) %>
+<% else %>
+  <%= render Primer::Beta::Button.new(**@system_arguments) do |button| %>
+    <% button.with_leading_visual_icon(icon: @icon) unless @icon.nil? %>
+    <% button.tooltip(text: content) if @icon_only == :when_narrow %>
+    <%= content %>
```

Comment:
Is it ok to allow arbitrary content in the button? Could we get by with a text-only attribute like `label:` instead?